### PR TITLE
Add loading state and disabled state when linking or when skipping

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- loading state for link button
- disabled state for skip button

### Changed

- disabled state for skip button

---

Ticket: https://jira.uitdatabank.be/browse/III-3793

Screenshot:
Schermafbeelding 2021-03-12 om 11 46 57

### CODE REVIEW

File: src/components/pages/manage/productions/create/index.js [hunk 1]
```diff
@@ -50,19 +50,31 @@ const Create = () => {
     await refetchSuggestedEvents();
   };
 
-  const { mutate: skipSuggestedEvents } = useSkipSuggestedEvents({
+  const {
+    mutate: skipSuggestedEvents,
+    ...skipSuggestedEventsMutation
+  } = useSkipSuggestedEvents({
```
Comment (geoffreydhuyvetters) [inline src/components/pages/manage/productions/create/index.js:56]: Can't we just use skipSuggestedEventsMutation.mutate everywhere? The mix of restructuring and renaming is a bit confusing imo.

File: src/components/pages/manage/productions/create/index.js [hunk 2]
```diff
@@ -89,6 +101,20 @@ const Create = () => {
...
+        createProductionWithEventsMutation.status,
+        mergeProductionsMutation.status,
+        addEventsByIdsMutation.status,
+      ].some((status) => status === QueryStatus.LOADING),
+    [
+      createProductionWithEventsMutation.status,
+      mergeProductionsMutation.status,
+      addEventsByIdsMutation.status,
+    ],
+  );
```
Comment (geoffreydhuyvetters) [inline src/components/pages/manage/productions/create/index.js:116]: We could also use something like this here.

https://react-query.tanstack.com/reference/useIsFetching

**Review by geoffreydhuyvetters — APPROVED** [review]

File: src/components/pages/manage/productions/create/index.js (hunk 2 above)
Comment (simon-debruijn) [reply src/components/pages/manage/productions/create/index.js:104]: I don't think we can use this one. Because it only counts the queries being executed, since mutations don't contain a key

File: src/components/pages/manage/productions/create/index.js (hunk 1 above)
Comment (simon-debruijn) [reply src/components/pages/manage/productions/create/index.js]: Changed it

File: src/components/pages/manage/productions/create/index.js (hunk 2 above)
Comment (geoffreydhuyvetters) [reply src/components/pages/manage/productions/create/index.js:104]: True!

**Review by geoffreydhuyvetters — APPROVED** [review]

File: src/components/pages/manage/productions/create/index.js [hunk 3]
```diff
@@ -50,19 +50,19 @@ const Create = () => {
     await refetchSuggestedEvents();
   };
 
-  const { mutate: skipSuggestedEvents } = useSkipSuggestedEvents({
+  const skipSuggestedEventsMutation = useSkipSuggestedEvents({
```
Comment (geoffreydhuyvetters) [inline src/components/pages/manage/productions/create/index.js:53]: To go a step further, we can just call this `skipSuggestedEvents` too, no need to suffix to it with 'Mutation'.

But we can do that in a different PR, let's merge this one :)

**Review by geoffreydhuyvetters — APPROVED** [review]
